content/blog/2021/12/first-year-of-kubewarden.md

```diff
@@ -0,0 +1,122 @@
+---
+title: First year of Kubewarden
+authors:
+    - Flavio Castelli
+date: 2021-12-22
+---
+
+Year 2021 is almost over. Let's take that as a chance to look back at what has been achieved during the 1st year of life of the Kubewarden project.
+
+Finally, I'll also talk about what we plan to do during the next one.
+
+{{< figure class="center" src="/images/year-progress.png" alt="Year Progress Bar">}}
+
+## 2021 Highlights
+
+### Project Announcement
+
+The Kubewarden project has been introduced to the masses for the 1st time during KubeCon Europe 2021.
+Rafael and I focused explained what lead us to rethink
+how Kubernetes policies could be written and distributed.
+The talk recording can be found [here](http://www.youtube.com/watch?v=Bhw5Qi78jj8), watch it to better understand Kubewarden's mission.
+
+During the remaining part of the year, Kubewarden talks featured at other conferences like KubeCon EU Security Day, Open Source Summit, Container Days, KubeCon NA Wasm Day, Kubernetes Community Days Italy and other live streaming events. What a year! 🤩
+
+
+### The `kwctl` Utility
+
+Shortly after KubeCon EU, we expanded the Kubewarden toolkit by releasing the [`kwctl`](https://github.com/kubewarden/kwctl/)
+utility.
+
+This is a command line tool aimed both at policy authors and Kubernetes operators.
+You can picture `kwctl` as a sort of `docker` for Kubewarden policies.
```

Comment:
Can we change what we are comparing it to? I conceptualized `kwctl` closer to `kubectl`.

Reply:
Done